src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -550,6 +550,39 @@ <h2><span jhiTranslate="artemisApp.exercise.details">Exercise Details</span></h2
                         <span>{{ programmingExercise?.coveredLinesRatio ? (programmingExercise?.coveredLinesRatio! * 100).toFixed(1) + ' %' : 'n.a.' }}</span>
                     </dd>
                 </div>
+                <div *ngIf="programmingExercise.isAtLeastEditor && programmingExercise.buildLogStatistics">
+                    <dt><span jhiTranslate="artemisApp.programmingExercise.buildLogStatistics.title">Build Log Statistics</span></dt>
+                    <dd>
+                        <table class="table table-striped">
+                            <thead>
+                                <tr>
+                                    <th jhiTranslate="artemisApp.programmingExercise.buildLogStatistics.numberOfBuilds"># Builds</th>
+                                    <th jhiTranslate="artemisApp.programmingExercise.buildLogStatistics.agentSetupDuration">Docker Setup</th>
+                                    <th jhiTranslate="artemisApp.programmingExercise.buildLogStatistics.testDuration">Test Execution</th>
+                                    <th jhiTranslate="artemisApp.programmingExercise.buildLogStatistics.scaDuration">Static Code Analysis</th>
+                                    <th jhiTranslate="artemisApp.programmingExercise.buildLogStatistics.totalJobDuration">Total Job Duration</th>
+                                    <th jhiTranslate="artemisApp.programmingExercise.buildLogStatistics.dependenciesDownloadedCount">Number of downloaded dependencies</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td>{{ programmingExercise.buildLogStatistics.buildCount ? programmingExercise.buildLogStatistics.buildCount : 0 }}</td>
```

Comment:
Thanks, very nice improvement! Sadly it does only work for the build count and the dependencies downloaded count since the other values have the `s` added.